server/core/runtime/models/shell_command_runner.go

```diff
@@ -154,11 +174,15 @@ func (s *ShellCommandRunner) RunCommandAsync(ctx command.ProjectContext) (chan<-
 
 		// We're done now. Send an error if there was one.
 		if err != nil {
-			err = errors.Wrapf(err, "running %q in %q", s.command, s.workingDir)
+			err = errors.Wrapf(err, "running %q in %q",
```

Comment:
Can we use `'%s'` rather than `%q`. This is because double quotes need to be escaped in the JSON log files, and therefore make them more difficult to read. Same below.